<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Customer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nameLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Address Line 1:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="addressLine1LineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Address Line 2:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="addressLine2LineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Zipcode:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="zipcodeLineEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>E-mail:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="emailLineEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Phone number:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="phoneNumberLineEdit"/>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>City</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Country:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="countryComboBox"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>State:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="stateComboBox"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>City:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="cityComboBox"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveBtn">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>countryComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Form</receiver>
   <slot>setStateModel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stateComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Form</receiver>
   <slot>setCityModel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nameLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>addressLine1LineEdit</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>137</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addressLine1LineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>addressLine2LineEdit</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addressLine2LineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>zipcodeLineEdit</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zipcodeLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>emailLineEdit</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>emailLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>phoneNumberLineEdit</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>phoneNumberLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>countryComboBox</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>153</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveBtn</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelBtn</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>384</y>
    </hint>
    <hint type="destinationlabel">
     <x>89</x>
     <y>367</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setStateModel()</slot>
  <slot>setCityModel()</slot>
  <slot>save()</slot>
 </slots>
</ui>
